tf/env/local/config-maps.tf

```diff
@@ -13,3 +13,16 @@ resource "kubernetes_config_map" "storage-bucket" {
     gcs_api_static_bucket_name = "api-assets"
   }
 }
+
+resource "kubernetes_config_map" "mediawiki_host_map" {
```

Comment:
I think that a config map like this should live in the k8s folder or maybe in a chart since that is also where the service names (and therefore the hosts) are defined.

I think only config maps that are containing data from TF resources should go in here.